tapsend/send.go

```diff
@@ -609,13 +611,19 @@ func PrepareOutputAssets(ctx context.Context, vPkt *tappsbt.VPacket) error {
 		// it's a zero value (tombstone) split output for the sender.
 		if vOut.Type.IsSplitRoot() {
 			vOut.Asset = splitCommitment.RootAsset.Copy()
+			vOut.Asset.LockTime = vOut.LockTime
+			vOut.Asset.RelativeLockTime = vOut.RelativeLockTime
+			vOut.Asset.ScriptKey = vOut.ScriptKey
```

Comment:
We weren't setting this before? 🤔

Reply:
I think they get set as part of `.Copy()`.